### PR TITLE
Replace weaviate provider with apache-airflow-providers-weaviate==1.3.0

### PR DESCRIPTION
**Pull request by sunank200**
- Replace weaviate provider with apache-airflow-providers-weaviate==1.3.0rc2
- Tested the DAG run [here](https://cloud.astronomer.io/clmkpupdk000401lpj28teo2t/deployments/clozirtvr4870661lvm82gj0n9c/overview)

closes: #136

### CODE REVIEW

File: airflow/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apache-airflow-providers-weaviate==1.0.0
+apache-airflow-providers-weaviate==1.3.0rc2
```
Comment (Lee-W) [inline airflow/requirements.txt:1]: Looks like [1.3.0](https://airflow.apache.org/docs/apache-airflow-providers-weaviate/stable/index.html) is there. Should we use it instead?

**Comment by vatsrahul1001** [comment]
Verified this LGTM, and I do not see any regression in sources and response

[Results](https://docs.google.com/spreadsheets/d/13cVqNikix82YjCPA4t0XaULg3XccBnvrQUmQa9VwgC0/edit#gid=1143451957)

File: airflow/requirements.txt (hunk 1 above)
Comment (sunank200) [reply airflow/requirements.txt:1]: Done

File: airflow/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apache-airflow-providers-weaviate==1.0.0
+apache-airflow-providers-weaviate==1.3.0
```
Comment (pankajastro) [inline airflow/requirements.txt:1]: Why are we imposing this restriction? Will it always result in the installation of version 1.3.0?
can we just add  apache-airflow-providers-weaviate?

Comment (sunank200) [reply airflow/requirements.txt:1]: We have a version for every library as you can see in the file. Are you recommending to remove version for all of them?

Comment (Lee-W) [reply airflow/requirements.txt:1]: For application (in this case), I think pinning it to some version is suggested.

**Review by Lee-W — APPROVED** [review]

**Pull request merged**